proposals/0000-keyboard-apis.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Keyboard events
+author: Harini Kannan, Stephen Peters
+date: 05/21/2019
+---
+
+# RFC0000: Keyboard APIs
+
+## Summary
+
+Components need the ability to handle key strokes and implement custom logic when interacting with a React Native app using Keyboard. This proposal outlines the basic props and events needed to enable this scenario.
+
+## Motivation
+
+There are several usecases where end users may interact with `react-native` applications using keyboard. These include:
+- iPads and Android tablet devices with attached keyboards or bluetooth keyboards
+- `react-native-windows` apps that will be deployed on PCs and laptop computers
```

Comment:
Xbox too! :D unless there are different apis planned for keyboard vs controller?

Reply:
There will be some more APIs for XYFocus that are specific to Xbox. But, you are correct - these APIs will also benefit gamepad invocation scenarios. I will add that in.